@navikt/ds-react/stepper/src/Stepper.tsx

```diff
@@ -0,0 +1,67 @@
+import "@navikt/ds-css/stepper/index.css";
+import cl from "classnames";
+import React, { createContext, forwardRef } from "react";
+import { uuid } from "../../util/src";
+import StepperStep from "./Step";
+
+export interface StepperProps extends React.HTMLAttributes<HTMLOListElement> {
+  children:
+    | React.ReactElement<typeof StepperStep>
+    | Array<React.ReactElement<typeof StepperStep>>;
+  activeStep: number;
+  colorful?: boolean;
+  arrow?: boolean;
+}
+
+export const StepContext = createContext({
+  activeStep: 0,
+});
+
+const Stepper = forwardRef<HTMLOListElement, StepperProps>(
+  (
+    {
+      children,
+      className,
+      activeStep,
+      colorful = false,
+      arrow = false,
+      ...rest
+    },
+    ref
+  ) => {
+    const steps = React.Children.toArray(children);
+    const stepsWithIndex = steps.map(
+      (step: React.ReactElement<typeof StepperStep>, index) => {
+        return (
+          <li key={uuid()} aria-current={index === activeStep && "step"}>
```

Comment:
Kanskje vi burde be bruker om å definere key, så man ikke ender opp med unødvendige og overraskende re-renders?

Reply:
Den forrige løsningen vil kanskje være bedre her siden man ikke trenger å sette en key i det hele tatt